client/src/components/browseUserRecipes/Recipes.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+  export let publishedBool;
+  export let recipes;
+  import Card from '@/components/common/Card.svelte';
+</script>
+
+<main>
+  <div class="recipes-container">
+    {#each recipes as { name, isPublished, avgRating, preparationTime, imageurl }}
+      {#if publishedBool === isPublished || publishedBool === undefined}
+        <Card title={name} option1={avgRating} option2={preparationTime} cover={imageurl} />
+      {/if}
+    {/each}
+  </div>
+</main>
+
+<style>
+  main {
+    height: 100%;
+  }
+
+  .recipes-container {
```

Comment:
css can be moved as a global class if you use it more than once but it would be even better if these two were just one component. Try doing that and if you get stuck we can work on it together